index.html

```diff
@@ -299,16 +299,62 @@ <h1>Sponsors</h1>
 
     <!--=================Card 6 Content Slide Navigation============-->
     <div id="myNav6" class="overlay6 tabcontent">
-
         <div class="flex-container">
             <a href="javascript:void(0)" class="closebtn" onclick="closeNav('myNav6')">&times;</a>
             <div class="image_card6"></div>
-            <div>
-                <h1>Project Showcase</h1>
-                <h3>More Information Coming Soon!</h3>
-            </div>
+            <h1>Project Showcase</h1>
+            <p>Here's a collection of the projects that were demoed from <b>UWB Hacks</b>.
```

Comment:
Let's keep the placeholders commented out until we are ready to write up projects.
```suggestion
            <h3>More Information Coming Soon!</h3>
            <!-- TODO: WRITE UP PROJECTS AFTER HACKATHON
            <p>Here's a collection of the projects that were demoed from <b>UWB Hacks</b>.
```